lib/galaxy_test/selenium/test_history_export.py

```diff
@@ -7,6 +7,10 @@
 class TestHistoryExport(SeleniumTestCase):
     ensure_registered = True
 
+    @classmethod
+    def handle_galaxy_config_kwds(cls, config):
+        config["enable_celery_tasks"] = False
```

Comment:
Non-integration tests shouldn't override this. Previously I don't think they even had this method. Is this definitely being used?

Reply:
> Is this definitely being used?

Yes, it does work, see the call stack in the screenshot to find where is being called:
![image](https://user-images.githubusercontent.com/46503462/206753876-fa1798da-e158-4a58-b7ed-9760bd8238e0.png)

But I see your point... this kind of turned into an integration test with this change... should I just move it to test/integration_selenium then?

Reply:
Since Celery is enabled by default in the test instance I need it to be disabled to keep testing the previous UI, so I decided to move it and convert it into an integration test for consistency.

Thank you John! let me know if I should approach it differently.